src/components/common/TransactionListRow.js

```diff
@@ -260,7 +260,7 @@ const getStyles = cacheStyles((theme: Theme) => ({
   }
 }))
 
-export const TransactionRow = connect((state: RootState, ownProps: OwnProps): StateProps => {
+export const TransactionListRow = connect((state: RootState, ownProps: OwnProps): StateProps => {
```

Comment:
Perfect. Since you renamed the component to TransactionListRow, you can create the raw style component as simply TransactionRow. TransactionRow can then be used by the FIO request scenes

Reply:
Done on latest commit